src/service/url-replacements-impl.js

```diff
@@ -367,11 +385,45 @@ export class UrlReplacements {
   }
 
   /**
+   * Registers click listener on documentElement that will expand the href of
+   * clicked anchors that will cause navigation (preventDefault has not been
+   * called) with additional support for CLICK_X & CLICK_Y based on clientX &
+   * clientY. If the target is within a shadowRoot, the x/y location is offset
+   * by the shadowRoot host's offset.
+   * Note that other click listeners may execute, which modify the href.  Because
+   * execution order of click listeners is non-deterministic across browsers,
+   * click listeners should not expect their modifications to be expanded.
+   * @private
+   */
+  registerClickListener_() {
+    if (!this.win_.document) {
+      dev.warn('no win.document?!  testing?');
+      return;
+    }
+    listen(this.win_.document, 'click', evt => {
```

Comment:
Should this be in capture phase, so it works when propagation is stopped?

Should it be on `document.documentElement`?